Gene.h

```diff
@@ -260,19 +260,19 @@ class Gene{
       int n = (*codonNum) % 3;
       int cdsIdx = i;
       switch(n){
-        case 0:
```

Comment:
I agree, the calculation in correct. In the original version when the program concludes, the location of the mutation was correct but the impact synonymous or non-synocymouns was reported from the wrong frame. (see before / after figures)
![gene original](https://cloud.githubusercontent.com/assets/16275969/15389381/7906351a-1dae-11e6-829a-09dd2c16420d.png "Original version")
![gene edited](https://cloud.githubusercontent.com/assets/16275969/15389386/84c0235c-1dae-11e6-8e75-df34f0fabb68.png "Edited version")